src/__tests__/pages/beta/key-stages/[keyStageSlug]/subjects/index.test.tsx

```diff
@@ -4,7 +4,7 @@ import curriculumApi from "../../../../../../node-lib/curriculum-api/__mocks__";
 import SubjectListingPage, {
   getStaticPaths,
   getStaticProps,
-} from "../../../../../../pages/beta/teachers/key-stages/[keyStageSlug]/subjects";
+} from "../../../../../../pages/beta/[viewType]/key-stages/[keyStageSlug]/subjects";
```

Comment:
is this `viewType` rather than `teacherViewType` because you anticipate a pupil view using the same URL structure as the teacher view?

Reply:
More or less, yes. The parameter at that place in the path could either be "teachers" or "pupils". After September it may no longer be a nextjs url paramter ... but would still be a a parameter in the sense that the url structure is assumed to be the same for routes nested within. Whether on teachers/pupils view. E.g. unit listing ... are you viewing it as a teacher or a pupil.